cli-tools/src/main/java/com/radixdlt/shell/RadixShell.java

```diff
@@ -178,9 +178,10 @@ public Node build() throws Exception {
         dataDir = new File(Files.createTempDirectory("radix-shell-node-").toString());
       }
 
-      customProperties.build().forEach((k, v) -> properties.set(k, v));
+      customProperties.build().forEach(properties::set);
 
       properties.set("db.location", dataDir.toString());
+      properties.set("db.location.node", new File(dataDir, "node").toString());
```

Comment:
In my opinion we shouldn't be introducing explicit configuration for e.g. the `db.location.node` field - it should just be a fixed path under `db.location`. Having the layout fixed under `db.location` means we don't have to worry about people changing this when people build backup scripts.

Side note - if we're introducing a new config, it should be included in the env_subst file so it works in Docker.

Reply:
Reworked to use the same config variable and fixed subdirectories under it.